code/modules/mob/living/simple_animal/abnormality/he/jangsan.dm

```diff
@@ -0,0 +1,288 @@
+	faction = list("hostile")
+	speak_emote = list("growls")
+	pixel_x = -16
+	base_pixel_x = -16
+	ranged = TRUE
+	maxHealth = 1200
+	health = 1200
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	see_in_dark = 10
+	stat_attack = HARD_CRIT
+	speed = 4
+	move_to_delay = 7
+	threat_level = HE_LEVEL
+	can_breach = TRUE
+	start_qliphoth = 3
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 45,
+						ABNORMALITY_WORK_INSIGHT = 45,
+						ABNORMALITY_WORK_ATTACHMENT = 45,
+						ABNORMALITY_WORK_REPRESSION = 45
+						)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+
+//Only done to non-humans, objects, and strong(er) agents
+	attack_sound = 'sound/abnormalities/jangsan/tigerbite.ogg'
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	melee_damage_lower = 40
+	melee_damage_upper = 60
+
+	ego_list = list(
+		/datum/ego_datum/weapon/maneater,
+		/datum/ego_datum/armor/maneater
+		)
+	gift_type =  /datum/ego_gifts/maneater
+
+//breach related
+	var/teleport_cooldown
+	var/teleport_cooldown_time = 120 SECONDS
+	var/strong_counter
+	var/weak_counter
+	pet_bonus = "meows" //saves a few lines of code by allowing funpet() to be called by attack_hand()
+	var/list/stats = list(FORTITUDE_ATTRIBUTE,
+			PRUDENCE_ATTRIBUTE,
+			TEMPERANCE_ATTRIBUTE,
+			JUSTICE_ATTRIBUTE)
+//attack vars
+	var/togglemovement
+	var/bite_cooldown
+	var/bite_cooldown_time = 8 SECONDS
+	var/chase_cooldown
+	var/chase_cooldown_time = 8 SECONDS
+	var/lure_cooldown
+	var/lure_cooldown_time = 120 SECONDS
+
+//speak_list + location + speak_list2
+	var/list/speak_list = list(";Hey guys im at ",
+			";Over here at ", ";Im in ")
+	var/list/speak_list2 = list(", let's have a pizza party!",
+			", i'll protect you!", ", let's work together!")
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/Initialize()
+	. = ..()
+	AbnoRadio()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, .proc/on_mob_death) // Hell
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/Move()
+	if (!togglemovement)
+		return FALSE
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/update_icon_state()
+	if(status_flags & GODMODE)	// Not breaching
+		icon_state = initial(icon)
+	else
+		icon_state = icon_aggro
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/Moved()
+	. = ..()
+	if(!(status_flags & GODMODE)) // Whitaker nerf
+		playsound(get_turf(src), 'sound/abnormalities/bigbird/step.ogg', 50, 1)
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/CanAttack(atom/the_target)
+	if(ishuman(the_target))
+		if(bite_cooldown > world.time)
+			return FALSE
+		var/mob/living/carbon/human/H = the_target
+		var/obj/item/bodypart/head/head = H.get_bodypart("head")
+		if(!istype(head)) // You, I'm afraid, are headless
+			return FALSE
+	togglemovement = TRUE
+	return ..()
```

Comment:
I understand that you copied it from big bird, but you might want to change it for attacking high level agents.
As is - it will never attack when on cooldown
Also what's up with random togglemovement?

Reply:
It can never "bite" high level agents - this is intentional. It will attack them normally. Bite is instant  death.

Reply:
That's the thing. It won't atack them normally when on cooldown because your check doesn't check for high level agents.

Reply:
I haven't noticed this issue when testing, but i'll look into it.